docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+allows you to deploy Trino and other applications in a repeatable manner across
+different types of infrastructure. This can range from deploying on your laptop
+using tools like `kind <https://kind.sigs.k8s.io>`_ and
+`Helm <https://helm.sh>`_, to running on a managed Kubernetes service on cloud
+services like
+`Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks>`_,
+`Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine>`_,
+`Azure Kubernetes Service <https://azure.microsoft.com/en-us/services/kubernetes-service>`_,
+and others.
+
+The fastest way to run Trino on Kubernetes is to use the
+`Trino Helm chart <https://github.com/trinodb/charts>`_. Helm is a package
+manager for Kubernetes applications that allows for simpler installation and
+versioning by templating Kubernetes configuration files. This allows you to
+prototype on your local or on-premise cluster and use the same deployment
+mechanism to deploy to the cloud to scale up.
+
+Prerequisites
+-------------
+
+* A Kubernetes cluster with a
+  `supported version <https://kubernetes.io/releases/>`_ of Kubernetes.
+
+  * If you don't have a Kubernetes cluster, you can
+    :ref:`run one locally using kind <running-a-local-kubernetes-cluster-with-kind>`.
+
+* The `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`_ binary.
+  Ensure that the version of ``kubectl`` adheres to the
+  `Kubernetes version skew policy <https://kubernetes.io/releases/version-skew-policy/>`_.
+
+* The `helm <https://helm.sh>`_ binary. Ensure that the version of ``helm``
+  adheres to the `Helm version skew policy <https://helm.sh/docs/topics/version_skew/>`_.
+
+Running Trino using Helm
+------------------------
+
+Run the following commands from the system with ``helm`` and ``kubectl``
+installed and configured to connect to your running Kubernetes cluster:
+
+#. Validate kubectl is pointing to the correct cluster by running the command:
+
+   .. code-block:: text
+
+      kubectl cluster-info
+
+   You should see output that shows the correct Kubernetes control plane
+   address.
+
+#. Add the Trino Helm chart repository to Helm if you haven't done so already.
+   This tells Helm where to find the Trino charts. You can name the repository
+   whatever you want, ``trino`` is a good choice.
+
+   .. code-block:: text
+
+       helm repo add trino https://trinodb.github.io/charts
+
+#. Install Trino on the Kubernetes cluster using the Helm chart. Start by
+   running the ``install`` command to use all default values and create
+   a cluster called ``my-trino-cluster``.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   This generates the Kubernetes configuration files by inserting properties
+   into helm templates. The helm chart contains
+   `default values <https://trinodb.github.io/charts/charts/trino/>`_
+   that can be overridden by a ``values.yaml`` file to update default settings.
```

Comment:
ideally dont use the filename "values.yaml" .. instead use one specific for the cluster ./ deployment you want to manage .. just think about how you would name it if you managed multiple clusters from the same mahcine and checked in the yaml files into a git repo

Reply:
also .. you can have many files passed in

Reply:
I want to avoid showing many files passed in option as it's not required to deploy Trino. They can learn that on the k8s docs for now until there's a common use case or pattern for deploying Trino that uses this. 

Rather than update the filenames as you've suggested, I'll add this note at the bottom of creating yaml values

```
.. _kubernetes-configuration-best-practices:

.. note::

   Although ``values.yaml`` is used to refer to the Kubernetes configuration
   file in this document, you should use clear naming guidelines for the cluster
   and deployment you are managing. For example, ``cluster-a-trino-etl.yaml``
   might refer to a Trino deployment for a cluster used primarily for 
   extract-transform-load queries deployed on the ``a`` Kubernetes cluster. See
   `Configuration Best Practices <https://kubernetes.io/docs/concepts/configuration/overview/>`_
   for more tips on configuring Kubernetes deployments.
```

Then I'll link to this location any time it's relevant. WDYT?

Reply:
Spoke with @mosabua , and we said this works, but we should change `values.yaml` to `example.yaml`.